README.md

```diff
@@ -49,6 +53,18 @@ We recommend using [Homebrew](https://brew.sh/), [rbenv](https://github.com/rben
   $ psql -c "CREATE DATABASE identity_idp_development;"
   $ psql -c "CREATE DATABASE identity_idp_test;"
   ```
+If you receive the following error (where _whoami_ == _your username_):
```

Comment:
These are great things to add, WDYT of adding a "troubleshooting" section at the bottom to catch all these errors? So that way skimming the README for instructions doesn't get as cluttered?

Reply:
Sure